src/nwb/NWBFile.cpp

```diff
@@ -30,24 +30,28 @@ NWBFile::NWBFile(const std::string& idText, std::shared_ptr<BaseIO> io)
 
 NWBFile::~NWBFile() {}
 
-void NWBFile::initialize()
+Status NWBFile::initialize()
```

Comment:
I think it may be useful to add a `bool` flag here for `overwriteIfExists`, but than can be a seperate low-priority issue